html/captive-portal/content/billing/authorizenet.js

```diff
@@ -35,10 +31,10 @@ console.log(secureData);
               for (var i = 0; i < response.messages.message.length; i++) {
                   console.log(response.messages.message[i].code + ": " + response.messages.message[i].text);
               }
-              alert("acceptJS library error!")
+              var $form = $('#payment-form');
+              $form.find('.payment-errors p').text('Unable to proceed with payment please contact your service provider');
```

Comment:
Actually, I propose you put a localized version of this error message inside the template since we only have one possible error message.

You could also improve the text. Ex:

>  We were unable to proceed your payment. Please contact your local support staff.